pages/documentation/migration-V3.md

```diff
@@ -0,0 +1,823 @@
+    <p><strong>What to do:</strong></p>
+    <ol>
+      <li>Search for any instances of <strong>thumb_down_off_alt</strong></li>
+      <li>Replace it with <strong>thumb_down_alt</strong></li>
+    </ol>
+  </div>
+  
+  <h4 class="usa-accordion__heading">
+    <button
+      class="usa-accordion__button"
+      aria-controls="m-a4"
+    >
+      If you have code older than USWDS 2.11.0
+    </button>
+  </h4>
+  <div id="m-a4" class="usa-accordion__content usa-prose">
+    <p><strong>We updated some settings defaults:</strong></p>
+    <ul>
+      <li>
+        <strong>new:</strong> $theme-alert-icon-size: 4<br>
+        <strong>old:</strong> 5
+      </li>
+      <li>
+        <strong>new:</strong> $theme-table-border-color: default<br>
+        <strong>old:</strong> ink
+      </li>
+      <li>
+        <strong>new:</strong> $theme-table-header-text-color: default<br>
+        <strong>old:</strong> ink
+      </li>
+      <li>
+        <strong>new:</strong> $theme-table-stripe-text-color: default<br>
+        <strong>old:</strong> ink
+      </li>
+      <li>
+        <strong>new:</strong> $theme-table-text-color: default<br>
+        <strong>old:</strong> ink
+      </li>
+    </ul>
+    <p><strong>What to do: </strong></p>
+    <ol>
+      <li>Search your codebase for any instances of <strong>the old theme</strong></li>
+      <li>Replace it with <strong>the new setting</strong></li>
+    </ol>
+  </div>
+  
+  <h4 class="usa-accordion__heading">
+    <button
+      class="usa-accordion__button"
+      aria-controls="m-a5"
+    >
+      If you have code older than USWDS 2.10.1
+    </button>
+  </h4>
+  <div id="m-a5" class="usa-accordion__content usa-prose">
+    <p><strong><span class="usa-tag bg-accent-warm-darker">Settings</span> We updated some settings defaults:</strong></p>
+    <ul>
+      <li>
+        <strong>Updated: </strong>$theme-breadcrumb-background-color: default<br>
+        <strong>Was: </strong>white 
+      </li>
+      <li>
+        <strong>Updated: </strong>$theme-alert-icon-size: 5 <br>
+        <strong>Was: </strong>4
+      </li>
+    </ul>
+    <p><strong>What to do:</strong></p>
+    <ol>
+      <li>Search your codebase for any instances of <strong>the old theme</strong></li>
+      <li>Replace it with <strong>the new setting</strong></li>
+    </ol>
+    <p><strong><span class="usa-tag bg-accent-cool-darker">Markup</span> We updated usa-footer__logo-heading to use a p instead of an h3.</strong> We improved the accessibility of the footer by converting a non-semantic heading into paragraph text.</p>
+    <p><strong>What to do: </strong></p>
+    <ol>
+      <li>TK</li>
+      <li>TK</li>
+    </ol>
+  </div>
+</div>
+
+### 3. Install the USWDS 3.0 package
+
+{:.border-top-2px.border-base-lighter.padding-top-1}
+1. In a terminal window, navigate to your project root. 
+2. Install USWDS 3.0 with **npm install @uswds/uswds --save --save-exact**
```

Comment:
This command returns an error. It should be `npm install @uswds/uswds  --save-exact` or `npm install @uswds/uswds --save-prod`